cms-derived-data/config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# all recids are placeholders, should be changed later to correct ones
+common_values:
+  abstract:
+    description: <p>Dataset in reduced NanoAOD format for education and outreach derived from the primary dataset in AOD format linked below.</p>
+  accelerator: CERN-LHC
+  authors: 
+    - CMS Open Data Group
+  collections:
+    - CMS-Derived-Datasets
+  control_number: 12365
+  dataset_semantics:
+  date_published: 2023
+  distribution:
+    formats:
+      - root
+      - nanoaod
```

Comment:
the format will be: nanoaod-poet, nanoaod-run1 and nanoaod-pf (https://github.com/cernopendata/data-curation/issues/212#issuecomment-1849938849)